master/reference/node/configuration.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Configuring calico/node
+---
+
+The `calico/node` container is primarily configured through environment variables.
+
+
+## Environment Variables
+
+| Environment | Description                              | Schema | Default |
+| ------------- | -------- | ------ | ----- |
+| NODENAME | Node ID. | string | |
+| NO_DEFAULT_POOLS | Prevents calico from creating a default pool if one does not exist. | boolean | false |
```

Comment:
I believe the default is an empty string :-(

Any non-empty value is taken to be true.